app/src/main/java/com/example/trempel/network/model/CategoryModelItem.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.trempel.network.model
+
+data class CategoryModelItem(
```

Comment:
I wonder if it can be marked as `internal`

Reply:
done